opensearch-operator/api/v1/opensearch_types.go

```diff
@@ -284,6 +284,11 @@ type ClusterStatus struct {
 // +kubebuilder:subresource:status
 // +kubebuilder:resource:shortName=os;opensearch
 // Es is the Schema for the es API
+// +kubebuilder:printcolumn:name="health",type="string",JSONPath=".status.health"
+// +kubebuilder:printcolumn:name="nodes",type="integer",JSONPath=".status.availableNodes",description="Available nodes"
```

Comment:
Both of these fields do not exist in the status object. You need to either remove them here or add them and logic to fill them to the operator.

Reply:
Will spend some time and try , able to add these fields in the status object but trying how to add the logic to the operator.

Reply:
pushed the changes.